README.md

```diff
@@ -40,6 +40,10 @@ container requires additional information and inputs:
 
 ## Changes
 
+Version 1.0.3 (2024-04-17)
+
+- Upgrade Dockerfile's base image to `ubuntu:20.04`, the same that is used in all other REANA components.
```

Comment:
Suggestion for the release news:

- Upgrade container base image to Ubuntu 20.04 LTS for consistency with the other REANA cluster components.